vector/src/main/java/im/vector/app/features/discovery/SettingsContinueCancelItem.kt

```diff
@@ -43,6 +46,7 @@ abstract class SettingsContinueCancelItem : EpoxyModelWithHolder<SettingsContinu
 
         continueText?.let { holder.continueButton.text = it }
         holder.continueButton.onClick(continueOnClick)
```

Comment:
should we always debounce? (maybe also a separate PR to always do it)

Reply:
Yes, maybe a dedicated PR will be better, also to use `fun View.onClick()` wherever it is possible